queries.yaml

```diff
@@ -432,6 +432,23 @@ post_process:
         - transit
         - water
 
+  # drop the names of small landuse polygons
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: landuse
+      start_zoom: 9
+      end_zoom: 12
+      properties: [name, operator, sport]
+      where: >-
+        pixel_area > area
+      geom_types: [Polygon, MultiPolygon]
```

Comment:
This mostly just leaves `kind`, and a bunch of properties which are derived from `kind`, which means things generally do get merged in the next step, even if it's just into a big MultiPolygon. I thought that's what the issue was asking for... perhaps @nvkelso could clarify?

If we're expecting polygons to get merged along seams then we might have to do some much more complex processing to bridge the gaps between them - on the example tile, most were disjoint.